docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -609,6 +610,12 @@ entry from the unit file takes precedence
 
 This key can be listed multiple times.
 
+### `SetWorkingDirectory=`
+
+Set the `WorkingDirectory` field of the `Service` group of the Systemd service unit file.
+Used to allow `podman kube play` to correctly resolve relative paths.
+Supported values are `yaml` and `unit` to set the working directory to that of the YAML or Quadlet Unit file respectively.
```

Comment:
What is the default?

Can we also allow for specifying absolute paths?

Reply:
> What is the default?

As far as Quadlet is concerned there is no default. If this key is not set, Quadlet will not do anything

> Can we also allow for specifying absolute paths?

We could. But, that does not provide any advantage over setting `WorkingDirectory` in the `Service` section. The whole point of this key is to have Quadlet calculate the working directory based on the location of the YAML or unit files

Reply:
Very fair point.  Could you mentioned `WorkingDirectory` in the docs as an alternative in the docs?

Reply:
I've added a new paragraph about explicitly setting this field